docs/07_deployment_view.adoc

```diff
@@ -37,50 +37,54 @@ when your infrastructure is more complex.
 === Infrastructure Level 1
 
 [role="arc42help"]
-****
-Describe (usually in a combination of diagrams, tables, and text):
-
-*  the distribution of your system to multiple locations, environments, computers, processors, .. as well as the physical connections between them
-*  important justification or motivation for this deployment structure
-* Quality and/or performance features of the infrastructure
-*  the mapping of software artifacts to elements of the infrastructure
-
-For multiple environments or alternative deployments please copy that section of arc42 for all relevant environments.
-****
-
-_**<Overview Diagram>**_
+[plantuml, "07_DeploymentView", png]
+----
+@startuml 07_DeploymentView_V0
 
-Motivation::
+node AzureServer{
+    node DockerContainer {
+        component RestApi
+    }
+}
 
-_<explanation in text form>_
+node UserDevice {
+    node WebBrowser {
+        component WebApp
+    }
+}
 
-Quality and/or Performance Features::
+database DatabaseServer {
+    component MongoDB
+}
 
-_<explanation in text form>_
+Actor User
 
-Mapping of Building Blocks to Infrastructure::
-_<description of the mapping>_
+cloud SOLID_POD
+cloud CarrierAPI
 
+RestApi -u-> MongoDB : "access information"
+WebApp -l-> SOLID_POD : "access user data"
+RestApi -d-> WebApp : "supply data"
+RestApi -r-> CarrierAPI : "calculate routes prices"
+WebApp <-d- User
 
-=== Infrastructure Level 2
+@enduml
+----
 
-[role="arc42help"]
-****
-Here you can include the internal structure of (some) infrastructure elements from level 1.
-
-Please copy the structure from level 1 for each selected element.
-****
-
-==== _<Infrastructure Element 1>_
-
-_<diagram + explanation>_
-
-==== _<Infrastructure Element 2>_
-
-_<diagram + explanation>_
-
-...
-
-==== _<Infrastructure Element n>_
-
-_<diagram + explanation>_
+.Motivation
+For development and testing the execution environment will be on each student's machine. But for the production phase it will be deployed in a Dockerized environment hosted on Azure. Our goal is to have a correct understanding of Docker, the infraestructure over which our system will be deployed.
```

Comment:
```suggestion
For development and testing the execution environment will be on each student's machine. But for the production phase it will be deployed in a Dockerized environment hosted on Azure. Our goal is to have a correct understanding of Docker, the infrastructure over which our system will be deployed.
```